lineman/app/components/thread_preview/thread_preview.haml

```diff
@@ -11,9 +11,8 @@
       %i.fa.fa-times>
       %span{translate: 'volume_levels.mute'}
 
-  %a.thread-preview__link{href: '/d/{{ thread.key }}'}
+  %a.thread-preview__link{lmo-href: '', lmo-href-model: 'thread'}
```

Comment:
I also added support in here so we can go to a specific route within a model path, ie
```
a{lmo-href: 'memberships', lmo-href-model: 'group'}
```

will go to the memberships page of the group.